.github/workflows/first_interaction.yml

```diff
@@ -16,32 +16,29 @@ jobs:
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
         issue-message: |
-          Thank you for contributing an issue!
+          Thank you for contributing an issue !
```

Comment:
Exclamation mark are like dot, so no space needed. Not sure why you added it on this pull request but that's fine

Reply:
I know Serge, but when this gets rendered they are barely visible https://github.com/dipy/dipy/issues/3397. Please remove them if you are uncomfortable with it.